app/components/comment.hbs

```diff
@@ -5,6 +5,12 @@
     <div class="post-description">on <span class="date">{{format-date @comment.createdAt}}</span>
     </div>
   </div>
+  <PaperButton
+  class="delete-icon {{if (eq @comment.user @getMe) "active"}}"
```

Comment:
Trying to add an if statement: if the owner of the comment is the same as me, then I want to add a class active (:visible icon) to the PaperButton.